src/pages/ServicePage/ServicePage.tsx

```diff
@@ -63,14 +74,15 @@ export const ServicePage = () => {
             <h5>OUR SERVICES</h5>
             <h1 className="fw-bold">We Build Software Solutionthat Solve Clients Business Challenges</h1>
             <p className="text-muted">Through True Rich Attended does no end it his mother since favourable real had half every him case in packages enquire we up ecstatic.</p>
-            <ButtonComponent text="Request A Quote"/>
+            <ButtonComponent text="Request A Quote" handleClick={handleShow}/>
+            <PopupPage open={show} close={handleClose}/> {/*****Popup-Page*****/}
           </Col>
           <Col md={6} className="services d-flex flex-column justify-content-center">
-            <Link to="/" className="service-link">Technical Support</Link>
-            <Link to="/" className="service-link">Development &#8594;</Link>
-            <Link to="/" className="service-link">AWS/Azure</Link>
-            <Link to="/" className="service-link">Consulting</Link>
-            <Link to="/" className="service-link">Information Technology</Link>
+            <a href="#Technical_Support" className="service-link" >Technical Support</a>
+            <a href="#Development" className="service-link">Development &#8594;</a>
+            <a href="#AWS/Azure" className="service-link">AWS/Azure</a>
+            <a href="#Consulting" className="service-link" id="Consulting">Consulting</a>
+            <a href="#Information_Technology" className="service-link">Information Technology</a>
```

Comment:
You could have used the same serviceListFeatures array to render these anchor tags